Assets/LeapMotionModules/Interaction/InteractionHand.cs

```diff
@@ -657,7 +667,7 @@ private struct HoverCheckResults {
     private void FixedUpdateContactCallbacks() {
       foreach (var interactionObj in _contactBehavioursLastFrame) {
         if (!_contactBehaviours.ContainsKey(interactionObj)
-          || !_brushBoneParent.gameObject.activeSelf) {
+         || !_brushBoneParent.gameObject.activeSelf) {
```

Comment:
watch out, even if activeSelf returns true the object can still be inactive if a parent was deactivated.  You probably want to use activeInHierarchy instead.